wp-content/themes/pub/wporg-learn-2020/template-parts/component-workshop-presenter.php

```diff
@@ -9,10 +9,11 @@
 
 $args = wp_parse_args( $args );
```

Comment:
If you give [`wp_parse_args` a default value,](https://developer.wordpress.org/reference/functions/wp_parse_args/) you can always assume that class exists on L16.

```suggestion
$args = wp_parse_args( $args, array( 'class' => '' ) );
```